docs/users/allowed-code.rst

```diff
@@ -0,0 +1,87 @@
+=============
+Code Allowed in WordPress/Largo Posts and Pages
```

Comment:
The allowed elements are [set in wp-includes/kses.php](https://codex.wordpress.org/Function_Reference/wp_kses_allowed_html), but I couldn't find any reference on wp.org detailing what they currently are. I'm fine with not listing this long list and understand the concern about inadvertently encouraging their use :)  The alternative probably is just to give examples of things disallowed, like scripts and iframes.